src/apps/project-collaboration/ProjectCollaboration.css

```diff
@@ -31,3 +31,43 @@
   line-height: 170%;
   padding: 10px 0;
 }
+
+.project-collaboration-request-header {
+  width: 50%;
```

Comment:
The `width: 50%` keeps collapsing things for me. Especially the text in the "copy link" button gets wrapped and then exceeds the button height.

Reply:
Fixed!